src/servers/main.cc

```diff
@@ -422,6 +429,54 @@ ParseFloatOption(const std::string arg)
   return std::stof(arg);
 }
 
+std::pair<int, std::vector<float>>
+ParseVGPUOption(const std::string arg, bool* malformed)
+{
+  std::vector<float> mem_limits_per_vgpu(0);
+
+  int delim_gpu = arg.find(";");
+  int delim_num_vgpus = arg.find(";", delim_gpu + 1);
+
+  // Check for 2 semicolons
+  if ((delim_gpu < 0) || (delim_num_vgpus < 0)) {
+    *malformed = true;
+    LOG_ERROR << "Cannot add virtual devices due to malformed TF-VGPU setting "
```

Comment:
Be specific and don't use terms (TF_VGPU) that don't necessarily mean anything to the user.
"--tf-add-vgpu argument requires format blah;blah;blah"